org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/authentication/TrivialAccessController.java

```diff
@@ -263,4 +265,83 @@ public TrivialAuthConfig withLoginPageInstalled(final boolean loginPageInstalled
       return this;
     }
   }
+
+  /**
+   * Checks if the authentication status type is authenticated by either a valid subject or a valid principal.
+   *
+   * @param authenticationStatusType
+   *     authentication status type
+   * @return {@code true} if authenticated. Otherwise {@code false}.
+   */
+  protected boolean isAuthenticatedRequest(AuthenticationStatusType authenticationStatusType) {
```

Comment:
Argument should be `AuthenticationStatus` instance instead.

Is this method expected to be overwritten? Otherwise, consider moving it to the AuthenticationStatus class itself.